api/refresh_token_logic.py

```diff
@@ -0,0 +1,69 @@
+from flask import request
+from user_agents import parse
+import requests
+import hashlib
+import time
+import os
+from typing import Optional
+from database.postgres.dto.device_info import DeviceInfoDTO
+
+class UserInfoService:
```

Comment:
This service does not belong in API layer. Move to BL and merge with UserService